src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -13,6 +13,8 @@
 public class WritableFileChooser extends JFileChooser {
 
 	private static final long serialVersionUID = -8600149638325315049L;
+	
+	private static final int minimumSpaceRequirement =  5000000;
```

Comment:
Constant names should be all upper case and separated with underscores (instead of camel-case).
Consider adding the postfix MB to the name.
There's an extra space after the equal that should be removed.